src/components/TopBar/components/SearchDismissOverlay/SearchDismissOverlay.scss

```diff
@@ -2,7 +2,7 @@
 @import '../../variables';
 
 $entry-iterations: 1;
-$backdrop-color: rgba(33, 43, 54, 0.4);
+$backdrop-color: rgba(color('ink'), 0.4);
```

Comment:
Can this use a custom property with a color(‘ink’) fallback?

Reply:
Theming of the search experience is out of scope for this piece of work, would it be okay to revisit this at a later time, if it becomes needed?